sql/moz-fx-data-shared-prod/telemetry_derived/desktop_retention_clients_v1/backfill.yaml

```diff
@@ -0,0 +1,9 @@
+2024-06-03:
+  start_date: 2021-01-01
+  end_date: 2024-06-01
+  reason: DENG-3187 - new table, need to backfill to 2021-01-01 for KPI reporting
+  watchers:
+  - mhirose@mozilla.com
+  - anicholson@mozilla.com
+  - wichan@mozilla.com
```

Comment:
I see @bani has done some work on this table, cc in case you want to be a watcher here.

Reply:
I asked Marlene to give me updates on Slack 😅

Reply:
I can add, @bani